test/Microsoft.AspNet.Mvc.Core.Test/Controllers/DefaultControllerFactoryTest.cs

```diff
@@ -257,7 +215,12 @@ private IServiceProvider GetServices()
 
         private static DefaultControllerFactory CreateControllerFactory(IControllerActivator controllerActivator = null)
         {
-            controllerActivator = controllerActivator ?? Mock.Of<IControllerActivator>();
+            var controllerActivatorMock = new Mock<IControllerActivator>();
+            controllerActivatorMock
```

Comment:
We should move the `DefaultControllerFactory_DisposesIDisposableController` test out in that case. We're testing the behavior of the activator, not the controller factory.